nixos/modules/system/activation/top-level.nix

```diff
@@ -276,6 +205,24 @@ in
       '';
     };
 
+    system.systemBuilderCommands = mkOption {
```

Comment:
Why do we have both this and `system.extraSystemBuilderCmds` ? they seem to do the exact same?

Reply:
This name is better.

I've looked at the definitions and they're not order sensitive, so we can go ahead and migrate.
- https://github.com/NixOS/nixpkgs/pull/460976

Could've done that half a year after this change, but better late than never.